mappings/net/minecraft/util/collection/DataPool.mapping

```diff
@@ -10,4 +10,9 @@ CLASS net/minecraft/unmapped/C_owfsqzhr net/minecraft/util/collection/DataPool
 	METHOD m_sepjgpgp wrapCodecToAllowEmpty (Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
 	CLASS C_eaytywro Builder
 		FIELD f_gfrmxqai entries Lcom/google/common/collect/ImmutableList$Builder;
+		METHOD m_djjyqyvx addData (Ljava/lang/Object;)Lnet/minecraft/unmapped/C_owfsqzhr$C_eaytywro;
+			ARG 1 data
+		METHOD m_flncuvzo addDataWithWeight (Ljava/lang/Object;I)Lnet/minecraft/unmapped/C_owfsqzhr$C_eaytywro;
```

Comment:
```suggestion
		METHOD m_flncuvzo addWeighted (Ljava/lang/Object;I)Lnet/minecraft/unmapped/C_owfsqzhr$C_eaytywro;
```
just a tad cleaner with `weighted` instead of `withWeight`